queryfilter/tests/test_textfilters.py

```diff
@@ -1,87 +1,104 @@
 from __future__ import absolute_import
 
+from queryfilter.tests.base_test_case import FilterTestCaseBase
+from test_app.models import Data
 from ..textfilters import (
     TextFullyMatchedFilter, TextPartialMatchedFilter,
     TextStartsWithMatchedFilter, TextEndsWithMatchedFilter
 )
 
 
-class TestTextFilterMixin(object):
-    def setup(self):
-        self.field_name_to_test = "name"
-        self.text_to_test = "name_example"
-        self.dicts = [
+class TextFilterTestBase(FilterTestCaseBase):
+
+    field_name_to_test = "name"
+    text_to_test = "name_example"
+
+    def get_model_class(self):
+        return Data
```

Comment:
This is fine, however how about use property here 🤔 it seems a typical usage scenario of property to me :p

Reply:
previous i try to use

abc.abstractmethod + property together and it fails orz. 

now I found !
abstractproperty